library/src/main/java/com/github/paolorotolo/appintro/AppIntro.java

```diff
@@ -17,78 +12,11 @@
 public abstract class AppIntro extends AppIntroBase {
     private static final String TAG = "AppIntro";
 
-    protected boolean skipButtonEnabled = true;
```

Comment:
Hi, short answer is "moved" not "removed". 

In the current master the skip button is only used inside the ```AppIntro``` but not in ```AppIntro2```. I moved the skip button code from ```AppIntro``` into the ```AppIntroBase``` class as the skip button could be part of both intro tour variants.